sdk/helper/consts/plugin_runtime_types.go

```diff
@@ -24,20 +22,11 @@ const (
 	PluginRuntimeTypeContainer
 )
 
-func (r PluginRuntimeType) String() string {
-	switch r {
-	case PluginRuntimeTypeContainer:
-		return "container"
-	default:
-		return "unsupported"
```

Comment:
Curious how enumer handles the defaults?

Reply:
The default will be `TypeName(number)`. For most of these, I tried to ensure that the default wasn't really used, or that the particular `String()` value wasn't relied on.